mmv1/provider/terraform_kcc.rb

```diff
@@ -87,76 +87,100 @@ def copy_common_files(output_folder, generate_code, generate_docs) end
     def guess_metadata_mapping_name(object)
       # Split the last import format by '/' and take the last part. Then use
       # the regex to verify if it is a value field in the format of {{value}}.
-      last_import_part = import_id_formats_from_resource(object)[-1].split('/')[-1].scan(/{{[[:word:]]+}}/)
+      last_import_part =
+        import_id_formats_from_resource(object)[-1].split('/')[-1].scan(/{{[[:word:]]+}}/)
       # If it is a value field, the length of last_import_part will be 1;
       # otherwise it'll be 0.
       # Remove '{{' and '}}' and only return the field name.
       last_import_part.first.gsub('{{', '').gsub('}}', '') if last_import_part.length == 1
     end
 
-    def is_server_generated_name(name, object)
-      has_computed_name_configured = object.custom_code.post_create == 'templates/terraform/post_create/set_computed_name.erb' if object.custom_code.post_create
+    def server_generated_name?(name, object)
+      if object.custom_code.post_create
+        has_computed_name_configured =
+          object.custom_code.post_create == 'templates/terraform/post_create/set_computed_name.erb'
+      end
       camel_case_name = name.camelize(:lower)
-      has_output_only_name = object.all_properties.any?{ |p| p.name == camel_case_name && p.output }
+      has_output_only_name =
+        object.all_properties.any? { |p| p.name == camel_case_name && p.output }
       has_computed_name_configured || has_output_only_name
     end
 
     def supports_conditions(iam_policy)
       request_type = iam_policy.iam_conditions_request_type
-      valid_request_types = ['QUERY_PARAM', 'QUERY_PARAM_NESTED', 'REQUEST_BODY']
-      return valid_request_types.include?(request_type.to_s)
+      valid_request_types = %w[QUERY_PARAM QUERY_PARAM_NESTED REQUEST_BODY]
+      valid_request_types.include?(request_type.to_s)
     end
 
     def get_resource_id_value_template(id_template, is_server_generated_name, object)
-      return nil if !is_server_generated_name
+      return nil unless is_server_generated_name
 
       if id_template.split('/').length == 1 && object.base_url != id_template
         raw_value_template = object.base_url
       end
-      return nil if raw_value_template == nil
 
-      value_template = raw_value_template + '/{{value}}'
```

Comment:
This change was manual- setting `value_template` didn't do anything so Rubocop wanted it removed. 116 and 122 are the same line (just adjusted to call `.nil?` automatically)

Reply:
Actually this line is necessary. I'm adding it back in https://github.com/GoogleCloudPlatform/magic-modules/pull/7123